content/chainguard/chainguard-images/working-with-images/image-stigs.md

```diff
@@ -0,0 +1,120 @@
+
+
+## What are STIGs?
+
+"STIG" is an acronym that stands for Secure Technical Implementation Guide. A STIG is akin to the implementation of a Security Requirements Guide (SRG) that a security administrator can go through to ensure that a given piece of software has been hardened against cybersecurity threats.
+
+A STIG is typically written by the developer or vendor of the given piece of software against a published DOD Security Requirements Guide (SRG). STIGs are presented in the XCCDF (Extensible Configuration Checklist Description Format), allowing them to be ingested into a SCAP-validated tool to validate that a given target is in compliance with them.
+
+After drafting the STIG, the vendor will submit it to the Defense Information Systems Agency (DISA), an agency within the DoD. One of DISA's responsibilities is publishing and maintaining STIGs on the [DoD Cyber Exchange website](https://public.cyber.mil/stigs/downloads/), and the process from a STIG being submitting to it being published by DISA can take years. As of this writing, DISA has published over 450 STIGs for a wide variety of software applications.
+
+
+## How STIGs can be used to harden images
+
+STIGs are typically published for hardware, firmware, and specific applications. However, in recent years containerization has grown in popularity and is now the modern way to deploy applications. Without STIGs to  presents a gap in terms of clear instructions on how to securely but there has historically been a gap when it comes to STIGs for 
+
+Chainguard produces hardened, minimal container images with each one containing few to zero CVEs. Because of this, they adhere to many compliance standards where there's a control for vulnerability or risk management, including [FedRAMP](https://www.fedramp.gov/) and [PCI DSS v4.0](https://www.pcisecuritystandards.org/document_library/?category=pcidss). 
+
+For many risk management frameworks, the system categorization will result in varying levels of hardening guidelines. Using FedRAMP as an example, FIPS-199 defines the security categorization which will result in a baseline set of controls (defined in NIST 800-53) that must be implemented.
+
+Because the NIST 800-53 controls are technology-neutral, the STIGs published by DISA provide technology-specific configurations on how to satisfy the applicable NIST 800-53 control set. As stated in the CM-6 (a) Requirement 1 of the FedRAMP System Security Plan:
+
+_“The service provider shall use the DoD STIGs to establish configuration settings; Center for Internet Security up to Level 2 (CIS Level 2) guidelines shall be used if STIGs are not available; Custom baselines shall be used if CIS is not available.”_
```

Comment:
This seems like the perfect place for a HTML blockquote (markdown this would start the line with `>`), but our blockquote element seems to create an infobox by default. I don't want to send you to a yakshaving place, I might create an issue for this for our backlog. Creating a new shortcode for this would not be difficult, it could be a regular quote format or could be our info box, but with a speech bubble instead of an I with a circle in it denoting information.

If what I'm saying here makes no sense then just resolve this comment and ignore. :)

References #1627

Reply:
No you're spot on. I initially had it set to use block quotes but didn't like the look of the boxes. This was very much my fast-and-easy solution. Thanks for creating the issue!